src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/Booking.kt

```diff
@@ -19,10 +19,10 @@ data class Booking(
   @Transient
   var singleTermSentence: SingleTermSentence? = null
 
-  fun getOrZero(adjustmentType: AdjustmentType, sentenceAt: LocalDate): Int {
+  fun getOrZero(adjustmentType: AdjustmentType, adjustmentsFrom: LocalDate): Int {
     if (adjustments.containsKey(adjustmentType) && adjustments[adjustmentType] != null) {
       val adjustments = adjustments[adjustmentType]!!
-      val adjustmentDays = adjustments.filter { it.appliesToSentencesFrom.isBeforeOrEqualTo(sentenceAt) }
+      val adjustmentDays = adjustments.filter { it.appliesToSentencesFrom.isBeforeOrEqualTo(adjustmentsFrom) }
         .map { it.numberOfDays }
       if (adjustmentDays.isNotEmpty()) {
         return adjustmentDays.reduce { acc, numberOfDates -> acc + numberOfDates }
```

Comment:
Is `numberOfDates` named correctly here? is this the totalNumberOfDays?

Reply:
I think the acc is the totalNumberOfDays in the reduce function. Thats just how it's named in the kotlin docs.